ironfish/src/rpc/routes/wallet/serializers.ts

```diff
@@ -157,6 +158,9 @@ export async function serializeRpcAccountStatus(
   account: Account,
 ): Promise<RpcAccountStatus> {
   const head = await account.getHead()
+  const isMultisigAccount = account.multisigKeys
+    ? isSignerMultisig(account.multisigKeys)
+    : false
```

Comment:
Should we also set this to true if `isHardwareSignerMultisig`? In the future we may also want to indicate that those accounts are Ledger accounts, but they're also multisig accounts

It's less clear to me how we should handle coordinator accounts 🤔

Reply:
yeah i was thinking maybe we want a `type` field instead? but i think that might push into the domain of your work with jason

Reply:
What would the different values of `type` be? Something like this?
```
- signer
- view-only
- multisig-signer
- multisig-coordinator
- ledger
- multisig-ledger
```

We're planning on refactoring how the wallet manages multisig identities, but I don't think we'll be adding or removing any types of accounts